pkg/ccl/crosscluster/logical/lww_row_processor_test.go

```diff
@@ -375,10 +375,8 @@ func TestLWWConflictResolution(t *testing.T) {
 	}
 
 	insertRow := func(rp RowProcessor, keyValue roachpb.KeyValue, prevValue roachpb.Value) error {
```

Comment:
could you sprinkle a few cases where prevValue does not match what's persisted and that the right error is returned?